src/pages/en/guides/server-side-rendering.mdx

```diff
@@ -121,6 +121,33 @@ The features below are only available at the page level. (You can't use them ins
 This is because these features modify the [Response headers](https://developer.mozilla.org/en-US/docs/Glossary/Response_header), which can't be modified after they're sent to the browser. In SSR mode, Astro uses HTML streaming to send each component to the browser as it renders them. This makes sure the user sees your HTML as fast as possible, but it also means that by the time Astro runs your component code, it has already sent the Response headers.
 :::
 
+### `Astro.cookies`
+
+This is a utility to read and modify a single cookie. It allows you to check, set, get and delete a cookie.
+
+Use `Astro.request.headers.get('cookie')` to get all cookies at once.
+
+See more details about [`Astro.cookies` and the `AstroCookie` type](/en/reference/api-reference/#astrocookies) in the API reference.
+
+The example below updates the value of a cookie for a page view counter.
+
+```astro title="src/pages/index.astro" {4,5,9}
+---
+let counter = 0
+
+if(Astro.cookies.has("counter")){
+  const cookie = Astro.cookies.get("counter")
+	counter = parseInt(cookie.value) + 1
```

Comment:
It's a little easier to do `cookie.number()` which parses it for you.

Reply:
Yeah, you meant `cookie.value.number()`, why not.

Reply:
It should just be `cookie.number()`. `cookie.value` is the raw string value. The definition of the return value of get() is here: https://github.com/withastro/astro/blob/9ef0c50427e658e0a6a4f20b14ff39c67fed76a6/packages/astro/src/core/cookies/cookies.ts#L37

Reply:
right, so it's an object API, did not realise that, then it's good to add it to the example